apps/condo/domains/common/utils/sberCloudFileAdapter.spec.js

```diff
@@ -0,0 +1,52 @@
+const ObsClient = require('esdk-obs-nodejs')
+const FOLDER_NAME = '__jest_test_api___'
+
+
+class SberCloudObsTest {
+    constructor (config) {
+        this.bucket = config.bucket
+        if (!this.bucket) {
+            throw new Error('SberCloudAdapter: S3Adapter requires a bucket name.')
+        }
+        this.obs = new ObsClient(config.s3Options)
+        this.folder = config.folder     
+    }
+
+    async uploadObject (name, text) {
+        const serverAnswer = await this.obs.putObject({
+            Bucket: this.bucket,
+            Key: `${this.folder}/${name}`,
+            Body: text,
+        })
+        return serverAnswer
+    }
+
+    async deleteObject (name) {
+        const serverAnswer = await this.obs.deleteObject({
+            Bucket: this.bucket,
+            Key: `${this.folder}/${name}`,
+        })
+        return serverAnswer
+    }
+}
+
+const initApi = () => {
+    const S3Config = process.env.SBERCLOUD_OBS_CONFIG ? JSON.parse(process.env.SBERCLOUD_OBS_CONFIG) : {}
+    if (S3Config.testBucket) {
+        S3Config.bucket = S3Config.testBucket
+    }
+    S3Config.folder = FOLDER_NAME
+    const API = new SberCloudObsTest(S3Config)
+    return API
+}
+
+describe('Sbercloud OBS API', () => {
+    test('Add File (and remove after add)', async () => {
```

Comment:
What about download? Is it working without credentials?

Reply:
I removed this test as this test depends on the environment and not everybody will use SberCloud to store files.
In fact, this test is only a check that we still have an access to the SbercloudObs
To test cloud availability better to put this to Prometheus or smth